src/components/PaymentLog.vue

```diff
@@ -1320,10 +1325,17 @@ export default {
     },
     getCalc() {
       this.watchCashPrice()
-
-      this.serial = ["product", "ap_cash_n_carry", "products"].includes(
-        this.salesLogForm.business_type_id?.slug
-      )
+      if (this.salesLogForm.business_type_id) {
+        if (
```

Comment:
can't this work here??

this.serial = ["product", "ap_cash_n_carry", "products"].includes(
        this.salesLogForm.business_type_id?.slug
      )

Reply:
The second if condition